src/os_win/os_mtx_cond.c

```diff
@@ -164,15 +160,11 @@ __wt_cond_signal(WT_SESSION_IMPL *session, WT_CONDVAR *cond)
  *	Destroy a condition variable.
  */
 void
-__wt_cond_destroy(WT_SESSION_IMPL *session, WT_CONDVAR **condp)
+__wt_cond_destroy(WT_SESSION_IMPL *session, WT_CONDVAR *cond)
 {
-	WT_CONDVAR *cond;
-
-	cond = *condp;
 	if (cond == NULL)
 		return;
 
 	/* Do nothing to delete Condition Variable */
 	DeleteCriticalSection(&cond->mtx);
-	__wt_free(session, *condp);
```

Comment:
We've removed the one place session is used.  I think we need an `WT_UNUSED` here now.